docs/development_environment.md

```diff
@@ -67,30 +82,34 @@ Visit the [Home Assistant repository](https://github.com/home-assistant/home-ass
 Once forked, setup your local copy of the source using the commands:
 
 ```bash
-$ git clone https://github.com/YOUR_GIT_USERNAME/home-assistant.git
-$ cd home-assistant
-$ git remote add upstream https://github.com/home-assistant/home-assistant.git
+git clone https://github.com/YOUR_GIT_USERNAME/home-assistant.git
+cd home-assistant
+git remote add upstream https://github.com/home-assistant/home-assistant.git
 ```
 
 ## Setting up virtual environment
 
 To isolate your environment from the rest of the system, set up a [`venv`](https://docs.python.org/3/library/venv.html). Within the `home-assistant` directory, create and activate your virtual environment.
 
 ```bash
-$ python3 -m venv .
-$ source bin/activate
+python3 -m venv .
+source bin/activate
 ```
+> If Python 3.6 installed as above in Linux Subsystem on Windows, use command:
```

Comment:
Please remove

Reply:
See comment here: https://github.com/home-assistant/developers.home-assistant/pull/173#discussion_r253554880

Reply:
See further comment here: https://github.com/home-assistant/developers.home-assistant/pull/173#discussion_r257337331

Reply:
@balloob Ubuntu for some reason choose to not include 3.5.3(Only 3.5.2) in Ubuntu 16.04 which is based on Debian Stretch. Since that's the version that's shipped with the Subsystem it would be preferable to have instructions to a workaround.